cli/commands/noop.go

```diff
@@ -0,0 +1,31 @@
+package commands
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/phase2/rig/cli/notify"
+)
+
+type Noop struct {
+	BaseCommand
+}
+
+func (cmd *Noop) Commands() []cli.Command {
```

Comment:
Was it a mistake to leave this command here?

Reply:
I left it in on purpose, but we can remove it if we want. It's a hidden command we can use to directly test notification behavior.